cluster/src/dunit/scala/io/snappydata/metrics/SnappyMetricsSystemDUnitTest.scala

```diff
@@ -319,51 +309,35 @@ class SnappyMetricsSystemDUnitTest(s: String)
         assertEquals("APP.TEST2", v)}
       if (containsWords(k, Array("TableMetrics.APP.TEST2", "isReplicatedTable"))) {
         assertEquals(false, v)}
-      if (containsWords(k, Array("TableMetrics.APP.TEST2", "sizeSpillToDisk"))) {
```

Comment:
Please explain these changes in the test as a PR description.

Reply:
Explained the changes in pr description